src/devtoolsPanel.ts

```diff
@@ -467,9 +504,11 @@ export class DevToolsPanel {
     private setCdnParameters(msg: {revision: string, isHeadless: boolean}) {
         if (msg.revision !== '') {
             this.config.isCdnHostedTools = true;
+            void vscode.commands.executeCommand('setContext', 'isCdnHostedTools', true);
```

Comment:
I think this should be awaited

Reply:
I think it should be fine as is since none of the following code really depends on this context being set. The only thing this affects is whether the command to enable/disable css mirroring is available to the user which I don't think awaiting or not awaiting will make much of a difference.